dair_pll/multibody_terms.py

```diff
@@ -534,8 +534,9 @@ def scalars_and_meshes(
                 self.plant_diagram.plant,
                 self.plant_diagram.model_ids)
 
-        for body_pi, body_id in zip(self.lagrangian_terms.pi(), all_body_ids):
-            body_scalars = InertialParameterConverter.pi_to_scalars(body_pi)
+        for body_pi, body_id in zip(self.lagrangian_terms.pi_cm(), all_body_ids):
```

Comment:
calls a `pi_cm` method on the `LagrangianTerms` class which doesn't exist in the commit (likely a renaming of `LagrangianTerms.pi(self)`)

Reply:
Good catch -- your guess is correct.  Propagated the name change with [this commit](https://github.com/DAIRLab/dair_pll/pull/4/commits/b1faed21be2e84611ab8d337f99960f9f76d9c4f).